docs/source/release/v6.13.0/framework.rst

```diff
@@ -10,61 +10,105 @@ Algorithms
-- Fixes a bug in  :ref:`algm-Divide` where units were not properly cleared when dividing two :ref:`ragged_workspace <Ragged_Workspace>` with identical Y-axis Units. This issue could cause errors in downstream data reduction workflows where the resulting workspace should be unitless. The fix ensures proper unit handling for :ref:`ragged_workspace <Ragged_Workspace>` division operations.
-- Fix issue with :ref:`GroupDetectors <algm-GroupDetectors-v2>` where the algorithm would freeze on large datasets while adding EventList to the Output Workspace. Testing with TOPAZ-50006 execution time was reduced from 2+ hours to ~30 seconds.
-- Fixes a bug in :ref:`algm-LoadAndMerge` where output workspaces were stored in the ADS when set not to. This caused the leaking of temporary workspaces in algorithms that call :ref:`algm-LoadAndMerge` as a child. Legacy behaviour can be achieved by passing ``StoreInADS=True`` to the algorithm.
-- Add `MandatoryValidator<OptionalBool>` to property declaration in Divide algorithm
-- :ref:`SaveNexusProcessed <algm-SaveNexusProcessed>`: fix SEGFAULT due to slab size to data size mismatch, on write of ragged workspaces.
+- :ref:`GenerateGroupingPowder <algm-GenerateGroupingPowder>` now correctly calculates the number of pixels in a group,
+  which may have been incorrect under certain conditions. This could lead to incorrect grouping of pixels in the output
+  workspace. The issue was caused by an incorrect handling of the pixel indices when calculating the groups. The fix
+  ensures that pixel indices are correctly handled, resulting in accurate grouping of pixels in the output workspace.
+- In the script generated by :ref:`GeneratePythonFitScript <algm-GeneratePythonFitScript>` titles for the subplots have
+  been replaced by meaningful legends so that labels of output subplots do not overlap.
+- The 'Unweighted least squares' cost function in :ref:`algm-Fit` now works as expected when compared with the scipy
+  library.
+
+  - As part of the fix a new parameter named ``IgnoreInvalidData`` defaulted to ``false`` has been introduced into
+    :ref:`CalculatePolynomialBackground <algm-CalculatePolynomialBackground>` and :ref:`algm-DirectILLTubeBackground`
+    algorithms so that it can be forwarded to the :ref:`algm-Fit` algorithm.
+  - In :ref:`algm-PoldiFitPeaks2D`, the ``IgnoreInvalidData`` parameter is defaulted to ``true`` when invoking
+    :ref:`algm-Fit` to preserve existing behaviour.
+  - The ``IgnoreInvalidData`` parameter defaulted to ``false`` has been introduced into
+    :ref:`algm-ReflectometryBackgroundSubtraction`.
+
+- ``InstrumentArrayConverter`` and ``PeakData`` utility classes used in peak integration algorithms have been moved into
+  a common module located at ``plugins.algorithms.peakdata_utils.py``. Import paths related to those classes must be
+  updated from ``from plugins.algorithms.IntegratePeaksSkew import InstrumentArrayConverter, PeakData`` into
```

Comment:
Is this line correct? from seems to be repeated.

To be honest the whole bullet point is difficult to understand. Can @RichardWaiteSTFC help with possibly rewording this?